src/pudl/settings.py

```diff
@@ -650,72 +650,99 @@ class EtlSettings(BaseSettings):
 
     @root_validator(pre=False)
     def raw_table_validation_ferc1(cls, field_values):
-        """Require the presence of raw FERC1 tables needed for PUDL table creation."""
+        """Require the presence of raw FERC1 tables needed for PUDL table creation.
```

Comment:
Suggested words

```py
"""Ensure that FERC to SQLite and PUDL ETL settings are internally self-consistent.

If FERC Form 1 tables are specified in both the :class:`FercToSqliteSettings` and
:class:`Ferc1Settings` within a single :class:`EtlSettings` object, ensure that any
FERC Form 1 tables required by the PUDL ETL will be produced by the FERC to
SQLite extraction. Otherwise raise a validation error.

To prevent this validation error, you can either remove one of the sets of tables (if
you aren't performing both steps of the ETL) or ensure that the specified sets of
tables are compatible.
"""
```